spec/system/candidate_interface/candidate_account_spec.rb

```diff
@@ -40,6 +40,14 @@
 
     when_i_signed_in_more_than_a_week_ago
     then_i_should_be_signed_out
+
+    when_i_click_the_signin_link
+    and_i_submit_my_email_address_in_uppercase
+    then_i_receive_an_email_with_a_signin_link
+
+    when_i_visit_the_signup_page
+    and_i_submit_my_email_address_in_uppercase
+    then_i_receive_an_email_with_a_signin_link
```

Comment:
@tvararu I added your system spec but changed the L50 to expect a signin link. Is this spec what you'd expect and does it cover the sign-in bug you described?

Reply:
👍